pkg/k8shandler/forwarding.go

```diff
@@ -41,6 +41,11 @@ func isForwardingEnabled(cluster *logging.ClusterLogging) bool {
 
 func (clusterRequest *ClusterLoggingRequest) generateCollectorConfig() (config string, err error) {
 
+	if clusterRequest.cluster == nil || clusterRequest.cluster.Spec.Collection == nil {
+		logger.Warnf("skipping collection config generation as 'collection' section is not specified in the CLO's CR")
```

Comment:
is this really a "warn" ? i think this is a debug statement...

Reply:
The second thing, I guess. The condition sounds good, makes sense to add a tech debt ticket @ewolinetz  ?